### ISSUE TITLE
Digital story nav number colours

### ISSUE THREAD

**Issue by Norvard** [wellcometrust/wellcomecollection.org#781]
On my Dell monitor, on normal contrast settings, I cannot pick up the numbers which are on the white cards. Lets match the grey which we use as a background for "coming soon" cards with the big number inside the other cards.

![screen shot 2017-04-12 at 10 55 35](https://cloud.githubusercontent.com/assets/23660297/24952281/e7ded0d8-1f6e-11e7-999a-eae5016d82de.png)

**Comment by jennpb** [comment]
Yup. As this is an accessibility issue, can we deal with this soon @davidpmccormick @gestchild ?
